### PR TITLE
Butter UI component Library PR

### PR DESCRIPTION
**Pull request by kshivam99**
Please review my code and comment suggestions

### CODE REVIEW

**Comment by Sumanbitw** [comment]
Great work Shivam!  The design is very clean and the components are well designed.

**Comment by soham-parab** [comment]
Creative name and a creative logo! Really liked the Success and Error alerts.

File: src/styles/Avatar.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+.md--avatar{
+    width: 6rem;
+    height: 6rem;
+}
+
+.lg--avatar{
+    width: 10rem;
+    height: 10rem;
+}
```
Comment (soham-parab) [inline src/styles/Avatar.css:20]: Please recheck if the third avatar is a perfect circle.

File: src/components/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { AiFillCloseCircle } from "react-icons/ai";
+import { useRouteMatch, Link } from "react-router-dom";
+
+function StackList({ name }) {
+  let { url } = useRouteMatch();
+
+  return (
+    <Link className="link stacklist" to={`${url}/${name}`}>
+      <h1>{name}</h1>
+    </Link>
```
Comment (Sumanbitw) [inline src/components/Sidebar.js:12]: Just try to be consistent with the spacing .Overall  great job ! Keep going.

**Review by Sumanbitw — COMMENTED** [review]
Very elegant components. Looks fabulous.

File: src/components/Landing.js [hunk 3]
```diff
@@ -1,27 +1,51 @@
...
+import React from "react";
+import { Link } from "react-router-dom";
 
 function Landing() {
-    return (
-        <div className="landing-main-content">
-        <div className="landing-content-section">
-          <h1 className="landing-heading">
-            Build Light, Deploy Faster with
-            <span style={{color:"#FEFE00"}}> Butter UI </span>
```
Comment (soham-parab) [inline src/components/Landing.js:9]: I would suggest reconsidering the yellow used for 'ButterUI', it is slightly harsh on the eyes.

**Review by soham-parab — CHANGES_REQUESTED** [review]
Awesome work! Please consider the suggested changes, thank you.

File: src/components/Sidebar.js (hunk 2 above)
Comment (kshivam99) [reply src/components/Sidebar.js:12]: Thanks, fixed it

File: src/styles/Avatar.css (hunk 1 above)
Comment (kshivam99) [reply src/styles/Avatar.css:20]: yes it is

File: src/components/Landing.js (hunk 3 above)
Comment (kshivam99) [reply src/components/Landing.js:9]: okay, Ill fix it

**Pull request merged**